docs/hack_operator_make_run.md

```diff
@@ -0,0 +1,274 @@
+
+  ```console
+  kubectl -n openshift-logging apply -f hack/lokistack_dev.yaml
+  ```
+
+* Now run the operator locally in your machine:
+
+  ```console
+  make run
+  ```
+
+  This will create `distributor`, `compactor`, `ingester`, `querier` and `query-frontend` components only.
+
+  Confirm that all are up and running for `deployments` using:
+
+  ```console  
+  kubectl -n openshift-logging rollout status deployment/<DEPLOYMENT_NAME>
+  ```  
+
+  where `<DEPLOYMENT_NAME>` is the name of the deployment and can be found using:
+
+  ```console
+  kubectl -n openshift-logging get deployments
+  ```
+
+  Confirm that all are up and running for `statefulsets` using:
+
+  ```console  
+  kubectl -n openshift-logging rollout status statefulset/<STATEFULSET_NAME>  
+  ```  
+
+  where `<STATEFULSET_NAME>` is the name of the statefulset and can be found using:
+
+  ```console
+  kubectl -n openshift-logging get statefulsets
+  ```
+  
+* If you want `lokistack-gateway` component [1] to be deployed then you need to create a gateway secret [2] for the operator. This can be done using:
+
+  ```code
+  kubectl -n openshift-logging create secret generic test1 \
+    --from-literal=clientID="<CLIENT_ID>" \
+    --from-literal=clientSecret="<CLIENT_SECRET>" \
+    --from-literal=issuerCAPath="<ISSUER_CA_PATH>"
+  ```
+  
+* Now create a LokiStack instance using:
+
+  ```console
+  kubectl -n openshift-logging apply -f hack/lokistack_gateway_dev.yaml
+  ```
+  
+* Edit the [main file](https://github.com/ViaQ/loki-operator/blob/master/main.go) to set the flag values to `true` and rerun the operator using:
+
+  ```console
+  make run
+  ```
+
+  This will create `distributor`, `compactor`, `ingester`, `querier`, `query-frontend` and `lokistack-gateway` components.
+
+* If you make some changes to the operator's code, then just stop the operator locally using `CTRL + C`, update the code and rerun the operator locally:
+
+  ```console
+  make run
+  ```
+
+  This saves time by not deploying the operator again and again.
+
+* When everything works fine, for the final testing deploy everything to the cluster using this [document](https://github.com/ViaQ/loki-operator/blob/master/docs/hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+### Cleanup
+
+* Stop the operator from running locally by pressing `CTRL + C`.
+
+* Cleanup the LokiStack instance, CRDs, deployments on the cluster using:
+
+  ```console
+  make uninstall
+  ```
+
+### Notes
+
+[1] `lokistack-gateway` is an optional component deployed as part of Loki Operator. It provides secure access to Loki's distributor (i.e. for pushing logs) and query-frontend (i.e. for querying logs) via consulting an OAuth/OIDC endpoint for the request subject.
+
+[2] The OIDC configuration expects `clientID`, `clientSecret` and `issuerCAPath` which should be provided via a Kubernetes secret that the LokiStack admin provides upfront.
```

Comment:
Can this work on Kind??? if so maybe it makes sense to invest in enabling that.  (if not, maybe remove from this document, it might fir and be important in other places)

Reply:
This doesn't work on kind. The above details are mentioned under `OpenShift` section